### DISCUSSION TITLE
我和我的冠軍女兒-女性也能創下傳奇

### DISCUSSION THREAD

**Discussion by Jamesmoneya** [1101-GS4719-C/Python-Community-GS4719-C#1438] [General]
這是一部改編真人真事的一部電影，非常的好看，除了有動人的親情描述、精采好看的摔角運動，也有包含發人省思的社會批判。這部電影主要是要顛覆父權至上，並且告訴觀眾在父權的壓力之下印度女性是有多麼的弱勢，電影的節奏跟一般的印度電影不太一樣，男主角阿米爾罕曾是國家代表隊，但是一些原因都沒有拿到冠軍，一天男主發現女兒也流淌著摔角的血脈，為了一圓為國家拿到冠軍的夢想，於是男主就把這個夢想強壓在女兒，完全不顧女兒自己到底心中想要甚麼夢想，但還好劇中男主那句「就給我一年」，讓我們看出來，其實他也不是過度的不聽女兒思想，心中還是有設停損點。

**Comment by slps101023** [comment]
父權的兩面性
男主角對女兒們的嚴格訓練表面上看是一種強加的控制，反映了父權社會中的「男性掌控一切」的心態。但從另一個角度來看，他的行動也挑戰了當地文化中對女性角色的狹隘定義。他沒有讓女兒們屈服於傳統命運，而是為她們開闢了一條不尋常的道路。

**Comment by ShuliANN** [comment]
這是一部改編自真人真事的電影，不僅好看，更充滿情感深度與社會意義。電影結合了動人的親情故事、令人熱血沸騰的摔角運動，以及引人深思的社會批判。透過故事的發展，這部電影挑戰了傳統父權的觀念，讓觀眾看見在父權壓力下，印度女性所面臨的種種不公與掙扎。

**Comment by wcyeee** [comment]
我也覺得這部電影很好看！除了電影對親情的描述很細膩之外，我也希望這部電影可以鼓勵現在世界各地因為當地的風俗被束縛的女性們，可以藉由這部電影獲得力量，勇敢做自己想做的事情！
